src/cljs/rems/administration/export_applications.cljs

```diff
@@ -0,0 +1,93 @@
+(ns rems.administration.export-applications
+  "Page for exporting applications."
+  (:require [re-frame.core :as rf]
+            [rems.administration.administration :refer [administration-navigator-container]]
+            [rems.atoms :as atoms]
+            [rems.collapsible :as collapsible]
+            [rems.dropdown :as dropdown]
+            [rems.flash-message :as flash-message]
+            [rems.spinner :as spinner]
+            [rems.text :refer [text text-format localize-time]]
+            [rems.util :refer [fetch post!]]))
+
+(rf/reg-event-fx
+ ::enter-page
+ (fn [{:keys [db]}]
+   {:db (-> (dissoc db ::form)
+            (assoc ::loading? true))
+    ::fetch-forms nil}))
+
+(defn- fetch-forms []
+  (fetch "/api/forms"
+         {:handler #(rf/dispatch [::fetch-forms-result %])
+          :error-handler (flash-message/default-error-handler :top "Fetch forms")}))
+
+(rf/reg-fx ::fetch-forms fetch-forms)
+
+(rf/reg-event-db
+ ::fetch-forms-result
+ (fn [db [_ forms]]
+   (-> (assoc db ::forms forms)
+       (dissoc ::loading?))))
+
+(rf/reg-sub ::forms (fn [db _] (::forms db)))
+
+(rf/reg-sub ::selected-form-id (fn [db _] (get-in db [::form :form-id])))
+(rf/reg-event-db ::set-selected-form-id (fn [db [_ form-id]] (assoc-in db [::form :form-id] form-id)))
+
+(rf/reg-sub ::loading? (fn [db _] (::loading? db)))
+
+(defn- cancel-button []
+  [atoms/link {:class "btn btn-secondary"}
+   "/administration/applications"
+   (text :t.administration/cancel)])
+
+(defn- export-button [form-id]
+  ;; Disabling <a> element is done using instructions from Bootstrap docs.
+  (let [disabled? (nil? form-id)]
```

Comment:
Hm, okay, apparently it is good practice to use question mark only with predicates and not boolean variables, will fix.

Reply:
some people do use it for variables too, especially for parameter names

Reply:
Right, it looks like the convention of using question mark also for booleans is used elsewhere in the code as well, so leaving it as it is.